### ISSUE TITLE
Api endpoint accepts parameters

### ISSUE THREAD

**Issue by vlad-kahoun** [yiedl/ethereum-unsigned-raw-transaction-oracle#3]
Api endpoint accepts From, To, Amount and other params necessary to create a transaction.

**Comment by udtrokia** [comment]
fn (from, to, value ) -> raw_transaction

**Comment by vlad-kahoun** [comment]
@udtrokia yes, we need only the first one. Signing will take place on the client's side.
